Packs/PANOSPolicyOptimizer/Integrations/PANOSPolicyOptimizer/PANOSPolicyOptimizer.py

```diff
@@ -32,15 +33,31 @@ def __init__(self, url: str, username: str, password: str, vsys: str, device_gro
         self.session = requests.Session()
 
     def session_post(self, url: str, json_cmd: dict) -> dict:
-        response = self.session.post(url=url, json=json_cmd, verify=self.verify)
+        response = self.session.post(url=url, json=json_cmd, verify=self.verify,
+                                     headers=self.session_metadata["headers"])
         json_response = json.loads(response.text)
         if 'type' in json_response and json_response['type'] == 'exception':
             if 'message' in json_response:
                 raise Exception(f'Operation to PAN-OS failed. with: {str(json_response["message"])}')
             raise Exception(f'Operation to PAN-OS failed. with: {str(json_response)}')
         return json_response
 
-    def login(self) -> str:
+    @staticmethod
+    def version_parse(version: str) -> float:
```

Comment:
This will results in version `1.9` being larger than version `1.10`.
Use `LooseVersion` (already imported in `CommonServerPython`) instead.